File: samples/WiEngineSkeletonProject_java/src/com/wiyun/engine/skeleton/FirstScene.java
package com.wiyun.engine.skeleton;

import android.graphics.Typeface;

import com.wiyun.engine.nodes.Director;
import com.wiyun.engine.nodes.Label;
import com.wiyun.engine.nodes.Scene;
import com.wiyun.engine.types.WYSize;

public class FirstScene extends Scene {
	public FirstScene() {
		WYSize s = Director.getInstance().getWindowSize();
	    Label label = Label.make("Welcome to WiEngine", 20, Typeface.BOLD, "DroidSans");
	    label.setPosition(s.width / 2, s.height / 2);
	    addChild(label);
	    
	    autoRelease(true);
    }
}

File: Windows/samples/WiEngineSkeletonProject/WiEngineSkeletonProject/FirstScene.cpp
#include "stdafx.h"

FirstScene::FirstScene() {    
    wyLabel* label = wyLabel::make("Welcome to WiEngine", SP(20), BOLD, WY_DEFAULT_FONT);
    label->setPosition(wyDevice::winWidth / 2, wyDevice::winHeight / 2);
    addChildLocked(label);
}

FirstScene::~FirstScene() {
}
